nibabel/streamlines/header.py

```diff
@@ -0,0 +1,19 @@
+
+class Field:
```

Comment:
Do you need this class?   How about a simple tuple of FIELD_NAMES and then use the strings to index the header dictionaries - I mean: 

```
header['nb_streamlines'] = something
```

rather than

```
header[Field.NB_STREAMLINES] = something
```